guide/payments/receive.md

```diff
@@ -0,0 +1,213 @@
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/enter-details-from.png"
+   retina = "/assets/images/guide/payments/receive/enter-details-from@2x.png"
+   alt-text = "Screen asking the receiver to enter the name of the sender"
+   caption = "The receiver can specify the amount that they are requesting. This could be shared in payment links which would auto-fill the amount for the sender."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/enter-details-purpose.png"
+   retina = "/assets/images/guide/payments/receive/enter-details-purpose@2x.png"
+   alt-text = "Screen asking the receiver to enter the purpose of the payment"
+   caption = "Sometimes the purpose can be determined by identifying who the payment is from. Be mindful of that, and consider easy ways to help the receiver add more context."
+   width = 250
+   height = 541
+%}
+</div>
+
+If your design’s goal is to get the receiver in the quickest time to share an address, you can choose to show them a new address straight away. The other properties can be left as optional fields. Consider the trade-off when optimizing this way, as there will be less useful contextual details for reports that you may want to provide in the future.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/enter-details-compact.png"
+   retina = "/assets/images/guide/payments/receive/enter-details-compact@2x.png"
+   alt-text = "Screen showing QR Code for the address and optional entry for other properties"
+   caption = "All properties editable on a single screen"
+   width = 250
+   height = 541
+%}
+
+</div>
+
+<!-- 
+Update interopability page
+Since there are many Bitcoin applications for people to choose from, its likely that the sender and receiver are not using the same one and they don't have the abilitiy to exchange payment details within the application itself. 
+
+The exchange of payment details then most commonly happens outside of the wallet application. This 
+
+Since bitcoin is a open system and has many payment applications built ontop of it, there is a [standard format](https://bitcoin.design/guide/foundations/wallet-interoperability/#payment-links) of "payment links" that most bitcoin applications use to share payment details.
+-->
+
+## Choosing the type of address
+There are currently [three different versions](https://bitcoin.design/guide/glossary/#address) of bitcoin addresses. In order of oldest to newest they are; Legacy, Script, and SegWit. It is best practice for new applications to support SegWit, the latest, by default.
+
+The most commonly used addresses are still the “legacy” addresses since Segwit addresses are only a few years old, and some older applications have yet to update.
```

Comment:
Source?

Reply:
I feel like we should recommend a particular address type here - SegWit with a fallback or switch option, I don't think we have enough information here nor that users will be able to decide for themselves what to use as we explained possible causes only on the surface (which is enough for the design guide imo), just we should maybe do it in `DO` recommend BC1 with the option to switch.

Reply:
Perhaps stated too strongly. Still, non SegWit transactions still make up a large enough amount of transactions that it may become an occasional problem.

- https://blockchair.com/bitcoin/charts/segwit-usage

Reply:
I saw some places the "Script" address is referred to as "Compatibility" or "SegWit Compatability" address. Perhaps we should consider this. I'd roll that into a bigger update of the glossary address section though.